src/amo/components/CollectionManager/index.js

```diff
@@ -184,6 +183,8 @@ export class CollectionManagerBase extends React.Component<
 
     const trimmedValue = value.trim();
 
+    const SLUG_MAX_LENGTH = 30;
```

Comment:
If you put it outside of the class, then you can `export` it and re-`import` in your test case.

Reply:
I was thinking so , but I thought that we are using it only in one assertion so.. . 

Making the changes. Thanks